pkg/transformers/openshift/openshift.go

```diff
@@ -0,0 +1,655 @@
+package openshift
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/runtime"
+	utilerrs "k8s.io/kubernetes/pkg/util/errors"
+	"k8s.io/kubernetes/pkg/util/sets"
+
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	"github.com/openshift/origin/pkg/generate/app"
+	"github.com/openshift/origin/pkg/generate/git"
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	dockerfileutil "github.com/openshift/origin/pkg/util/docker/dockerfile"
+	"github.com/openshift/origin/third_party/github.com/docker/libcompose/project"
+
+	// Install OpenShift APIs
+	_ "github.com/openshift/origin/pkg/build/api/install"
+	_ "github.com/openshift/origin/pkg/deploy/api/install"
+	_ "github.com/openshift/origin/pkg/image/api/install"
+	_ "github.com/openshift/origin/pkg/route/api/install"
+	_ "github.com/openshift/origin/pkg/template/api/install"
+)
+
+type TransformData struct {
```

Comment:
this can be private because it is not supposed to be used outside this package `transformData`